docs/0.5/howto/removing_a_splinter_circuit.md

```diff
@@ -0,0 +1,366 @@
+# Removing a Splinter Circuit
+
+<!--
+  Copyright 2018-2021 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+This guide explains alternative approaches to delete a circuit, disbanding and
```

Comment:
change to "for deleting" (currently says "to delete a circuit", which doesn't fit the rest of the sentence)

Reply:
Fixed up this first paragraph overall